docs/src/polynomial.md

```diff
@@ -1084,4 +1084,8 @@ U, z = polynomial_ring(R, "z")
 h = rand(U, 3:3, -1:2, -10:10)
 ```
 
+### Ring homomorphisms
 
+```@docs
+hom(::PolyRing, ::NCRing, ::Any, ::RingElem)
```

Comment:
```suggestion
hom(::PolyRing, ::PolyRing, ::Any, ::RingElem)
```
Otherwise, I get
```
Error: no docs found for 'hom(::PolyRing, ::NCRing, ::Any, ::RingElem)' in `@docs` block in src/polynomial.md:1089-1091
```@docs
hom(::PolyRing, ::NCRing, ::Any, ::RingElem)
```